### PR TITLE
Todo

### PR DESCRIPTION
**Pull request by AnkitAegis**
isDeleted column added in schema

### CODE REVIEW

**Review by aryan-aegis — APPROVED** [review]

File: src/controllers/todoController.js [hunk 1]
```diff
@@ -13,49 +13,51 @@ const createTodo = async (req, res) => {
 
     res.status(200).send({ newTodo, message: 'todo created' })
   } catch (e) {
-    res.status(401).send({ message: 'Not fullfilled', error: e.message })
+    res.status(500).send({ message: 'Not fullfilled', error: e.message })
```
Comment (am9760) [inline src/controllers/todoController.js:16]: Try to set as many Logic Error Messages as possible for User

File: src/controllers/todoController.js [hunk 2]
```diff
@@ -13,49 +13,51 @@ const createTodo = async (req, res) => {
...
       }
     })
 
     res.status(200).send({ userDocs, message: 'get todo done' })
   } catch (e) {
-    res.status(401).send({ message: 'Not fullfilled', error: e })
+    res.status(500).send({ message: 'Not fullfilled', error: e })
   }
 }
 const patchTodo = async (req, res) => {
```
Comment (am9760) [inline src/controllers/todoController.js:57]: Use CRUD Name, not REST Method Names

File: readme.md [hunk 3]
```diff
@@ -1 +1,205 @@
-##express-assignment
\ No newline at end of file
+# express-assingment
```
Comment (am9760) [inline readme.md:1]: Nice Readme

File: src/controllers/todoController.js [hunk 4]
```diff
@@ -13,49 +13,51 @@ const createTodo = async (req, res) => {
 
     res.status(200).send({ newTodo, message: 'todo created' })
   } catch (e) {
-    res.status(401).send({ message: 'Not fullfilled', error: e.message })
+    res.status(500).send({ message: 'Not fullfilled', error: e.message })
   }
 }
 
 const getSingleTodo = async (req, res) => {
   let data = req.params;
```
Comment (am9760) [inline src/controllers/todoController.js:21]: Payload validation can be added [Optional]

File: src/controllers/todoController.js (hunk 1 above)
Comment (am9760) [reply src/controllers/todoController.js:16]: Also don't throw server errors directly to client, just show simple user-friendly messages

**Review by am9760 — COMMENTED** [review]
@AnkitParte @TejushAegis 
Minor changes, Overall looks Good

**Pull request merged**